io-engine/src/bdev/nexus/nexus_bdev_rebuild.rs

```diff
@@ -114,7 +115,11 @@ impl<'n> Nexus<'n> {
             }),
         }?;
 
-        self.create_rebuild_job(&src_child_uri, &dst_child_uri)
+        // Get child' rebuild log.
+        let log = self.lookup_child(child_uri).and_then(|c| c.rebuild_log());
```

Comment:
If we are in start_rebuild path via add_child call, then I believe we shouldn't find the rebuild log here. If we come via online_child path we'd find it. Can we internally call `drop_rebuild_log` as soon as we return the log handle here? Instead of few lines down?